server/custom_vms.go

```diff
@@ -91,16 +91,18 @@ func (lc *localNetwork) waitForCustomVMsReady(ctx context.Context) error {
 	println()
 	color.Outf("{{blue}}{{bold}}waiting for custom VMs to report healthy...{{/}}\n")
 
-	hc := lc.nw.Healthy(ctx)
-	select {
-	case <-lc.stopc:
-		return errAborted
-	case <-ctx.Done():
-		return ctx.Err()
-	case err := <-hc:
-		if err != nil {
-			return err
+	healthyCtx, healthyCtxCancel := context.WithCancel(ctx)
```

Comment:
I think you do not need this. Healthy is already taking into account stop events. You can ignore lc.stopc.